jobs/openshift/cluster/integreatly/osd-cluster-integreatly-uninstall.yaml

```diff
@@ -0,0 +1,56 @@
+---
+- job:
+    name: osd-cluster-integreatly-uninstall
+    display-name: 'OSD Cluster Integreatly Uninstall'
+    project-type: pipeline
+    concurrent: false
+    parameters:
+      - string:
+          name: 'clusterName'
+          default: ''
+          description: '[REQUIRED] The name of the target cluster to uninstall Integreatly against'
+      - string:
+          name: 'uninstallationGitBranch'
+          default: 'master'
+          description: '[REQUIRED] The name of the git branch to be used for the Integreatly uninstaller'
+      - string:
+          name: 'routerShard'
+          default: ''
+          description: '[REQUIRED] Router shard of the targeted cluster'
+      - string:
+          name: 'openShiftToken'
+          default: ''
+          description: '[REQUIRED] Openshift token for cluster-admin user'
+    dsl: |
+        node('cirhos_rhel7') {
+          stage('Uninstall Integreatly') {
+            wrap([$class: 'AnsiColorBuildWrapper', colorMapName: "xterm"]) {
+                ansibleTower(
+                  towerServer: 'Dev Tower',
+                  jobTemplate: 'Integreatly_Bootstrap_and_Uninstall_[OSD]',
+                  templateType: 'workflow',
+                  importTowerLogs: true,
+                  removeColor: false,
+                  verbose: true,
+                  extraVars: """---
+                    tower_environment: dev
+                    rhsso_namespace: openshift-sso
```

Comment:
Not sure about this line...just stole it from the template definition.

Reply:
Me neither, probably ask PaulMc or Steven.

Reply:
Or leave as is, it does not seem that we will be using a different value for rhsso namespace, but still, better to know why it is there.